index.js

```diff
@@ -127,25 +145,22 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventory = userData[userIdentification].inventory;
+    let cardIDs = Object.keys(userInventory);
```

Comment:
You were onto something earlier with `Object.values()`. When you use `Object.keys()`, you get an array of the keys in an object. When you use `Object.values()`, you get an array of the values in an object.

Note that we never actually use the cardId for anything -- we only ever use it to access the object value by doing `object[objectKey]`. That's a sign that maybe we want to be dealing with values instead of keys. Can you see how this would work if you used `Object.values()` instead of `Object.keys()`?